src/concrete/ml/quantization/quantized_module.py

```diff
@@ -577,6 +577,7 @@ def compile(
         p_error: Optional[float] = None,
         global_p_error: Optional[float] = None,
         verbose: bool = False,
+        statuses: Optional[Dict[str, str]] = None,
```

Comment:
I don't feel like statuses is very talkative. Should be referring to the input I think. `input_statuses` or `input_encryption` where the user passes booleans in a list following the order of the input in the forward call? so that you don't have to specify onnx node names.